clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -745,6 +747,9 @@ public synchronized void subscribePartition(
       Optional<PubSubPosition> pubSubPosition) {
     activeReplicaCount.incrementAndGet();
     LOGGER.info("Bootstrap replica: {}. Active replica count in SIT: {}", topicPartition, activeReplicaCount.get());
+    if (isCurrentVersion()) {
+      currentVersionResubscribeSet.add(topicPartition.getPartitionNumber());
```

Comment:
This is called by helix thread, SIT thread will also modify this set.  We might not need to maintain this set.